NEWS.md

```diff
@@ -7,6 +7,9 @@
 * Fixed an issue where calling `skip()` outside of an active test could
   cause an unexpected error (@kevinushey, #2039).
 
+* New `skip_unless_r()` to skip running tests on unsuitable versions of R, e.g.
```

Comment:
After you merge/rebase can you please move up to the dev version section? Also note that we're switching our style to not line wrap and not to have empty lines between bullets.